README.md

```diff
@@ -9,32 +9,17 @@ Contribute
 In order to contribute to this repository, extend code and tests 
 as appropriate and use clean commits.
 
-To setup your dev environment,
```

Comment:
this environment is still needed for development; this should be reflected in the README